Sources/Danger/GitDSL.swift

```diff
@@ -24,41 +18,37 @@ public struct Git: Decodable, Equatable {
     public let deletedFiles: [File]
 }
 
-// MARK: - GitCommit
-
-/// A platform agnostic reference to a git commit.
-public struct GitCommit: Decodable, Equatable {
-    // MARK: - Properties
+extension Git {
+    /// A platform agnostic reference to a git commit.
+    public struct Commit: Decodable, Equatable {
+        /// The author of a commit.
+        public struct Author: Decodable, Equatable {
```

Comment:
<!--
  0 failure: 
  1 warning:  Types should be n...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitDSL.swift;
  Line: 25;
-->

- :warning: Types should be nested at most 1 level deep (`nesting`)